API/DataModel/DataContext.cs

```diff
@@ -25,6 +30,23 @@ protected override void OnModelCreating(ModelBuilder builder)
 
             builder.Entity<Role>().HasMany(r => r.UserRoles).WithOne(r => r.Role)
                                   .HasForeignKey(r => r.RoleId).IsRequired();
+
+            //clustered keys
+            builder.Entity<OrderToServicePrice>()
```

Comment:
w ten sposób modelujesz wiele do wielu? bo ja zawsze robię tak jak wyżej za pomocą fluent api i encji pośredniej.

Reply:
To jest definicja tabeli, gdzie klucz podstawowy jest klastrem dwóch kolumn. Z tą encją nic nie robiłem więcej ale jest to potrzebne żeby powstała struktura bazy danych przy migracji.